--- gnustep-gui-0.23.1/Source/NSBitmapImageRep+GIF.m.orig	2013-04-01 23:02:18.757066893 +0100
+++ gnustep-gui-0.23.1/Source/NSBitmapImageRep+GIF.m	2013-04-01 23:06:54.031050809 +0100
@@ -142,7 +142,11 @@
     }
 
   gs_gif_init_input_source(&src, imageData);
+#if GIFLIB_MAJOR >= 5 
+  file = DGifOpen(&src, gs_gif_input, NULL);
+#else
   file = DGifOpen(&src, gs_gif_input);
+#endif
   if (file == NULL)
     {
       /* we do not use giferror here because it doesn't
@@ -216,7 +220,13 @@
 
   /* open the image */
   gs_gif_init_input_source(&src, imageData);
+
+#if GIFLIB_MAJOR >= 5 
+  file = DGifOpen(&src, gs_gif_input,NULL);
+#else
   file = DGifOpen(&src, gs_gif_input);
+#endif
+
   if (file == NULL)
     {
       /* we do not use giferror here because it doesn't
